README.md

```diff
@@ -16,7 +16,7 @@ delimiter sequence using the `-d` option.
 Using the arrow keys will move the selected field. The interface supports
 several Emacs and Vi like key bindings, consult the man page for further
 reference. Pressing the return key will invoke the yank command and write the
-selected field to its `stdin`. The yank command defaults to xsel [[1](#fn1)] but could be
+selected field to its `stdin`. The yank command defaults to [xsel][1] but could be
```

Comment:
Just call it `[xsel]` since the same reference can be used at the end to specify the actual URL. Once done, remove all previous commits.